bin/slss.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+// TODO (BREAKING): Remove this file with next major release
+
+'use strict';
+
+require('../lib/utils/logDeprecation')(
+  'DEPRECATED_ALIAS_SLSS',
```

Comment:
Let's not use `DEPRECATED_` prefixes in deprecation codes. They seems superfluous. I guess just `SLSS_CLI_ALIAS` would make a great name